lib/transbank/sdk/patpass/patpass_by_webpay/transaction.rb

```diff
@@ -0,0 +1,73 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      class Transaction
+        extend Transbank::Utils::NetHelper
+        CREATE_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        COMMIT_TRANSACTION_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+        TRANSACTION_STATUS_ENDPOINT = 'rswebpaytransaction/api/webpay/v1.0/transactions'
+
+        class << self
+
+          def create(buy_order:, session_id:, amount:, return_url:, options: nil)
+
+            api_key = options.api_key || default_integration_params[:api_key]
```

Comment:
here you need use `&` as `options&.api_key` because if options is null you will have an error.

Reply:
Ready! Safe navigation operator applied